lib/rails/generators/alchemy/install/templates/elements.yml.tt

```diff
@@ -1,6 +1,6 @@
-# == In this configuration you setup Alchemy´s element layouts.
+# == In this configuration, you set up Alchemy's element layouts.
 #
-# For further informations please see http://guides.alchemy-cms.com/create_elements.html
+# For further information please see http://guides.alchemy-cms.com/edge/elements.html
```

Comment:
I'm not sure about the link to the edge guides. Unfortunately we don't have a generic link to guide pages. Every link is versioned. But leaving the `edge` link here means we need to update this link for every new version...

Any thoughts @mamhoff @robinboening ?